packit_service/worker/helpers/build/build_helper.py

```diff
@@ -754,3 +755,22 @@ def report_status_to_configured_job(
             links_to_external_services=links_to_external_services,
             update_feedback_time=update_feedback_time,
         )
+
+    def notify_about_failure_if_configured(self):
+        """
+        If there is a failure_comment_message configured for the job,
+        post a comment and include the configured message. Do not post
+        the comment if the last comment from the Packit user is identical.
+        """
+        if (
+            configured_message := self.job_config.notifications.failure_comment_message
+        ) is None:
+            return
+
+        full_message = (
+            f":exclamation: **One of the {self.comment_job_name} failed for commit "
+            f"{self.db_project_event.commit_sha}**\n\n---\n\n{configured_message}"
```

Comment:
I was thinking about whether to rather provide these variables so that users can utilise them in the configuration themselves, WDYT?

Reply:
I think it would be easier to use/tweak if we users could define the full message. We can provide a good copy-pastable example (e.g. with the content you've just provided) so that every single user does not need to invent that.

Reply:
ok, sounds good to me